Olapic/angular-q-all-settled
# Add karma as an explicit dependency

#### What does this PR do?
Add karma as an explicit dependency to avoid issues while `npm install` process is running.
As I read in the logs, the bigger version dependency for karma is ~0.12.0 (some another dependecy are using >=0.9.0, so I added ~0.12.0 in the `package.json` file.

#### How should this be manually tested?
* Clone this repo
* Run `npm install`
Before adding karma, you should get some errors.
Those errors will be fixed after adding karma.

#### Any background context you want to provide?
Those last months `npm` team released some updates for `npm` that remove the implicit dependency installation. This error could be related to this thing, or probably, because some devs have installed karmar as a global dependency.

#### What are the relevant tickets?
No ticket.
